src/D3MHub.sol

```diff
@@ -490,6 +503,8 @@ contract D3MHub {
         _pool.preDebtChange("reap");
         uint256 assetBalance = _pool.assetBalance();
         (, uint256 daiDebt) = vat.urns(ilk, address(_pool));
+        daiDebt += _fix(ilk, address(_pool));
```

Comment:
There is another way to do this which is first executing `_fix`, then calling vat.urns and get the `daiDebt` corrected.
We could also change `_fix` to fixed daiDebt and in this case we would save two warmed SLOADs.
For `exec` though is better to get the diff as we are adding up to the `Art`. Conceptually it shouldn't be correct to replace the `Art` by the `art` of one CDP regardless it is the unique one of this type.

Reply:
As I said in the other comment, I'm not too concern about gas optimizations here.